staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -133,6 +136,17 @@ type KubeSchedulerProfile struct {
 	// is scheduled with this profile.
 	SchedulerName *string `json:"schedulerName,omitempty"`
 
+	// PercentageOfNodesToScore is the percentage of all nodes that once found feasible
+	// for running a pod, the scheduler stops its search for more feasible nodes in
+	// the cluster. This helps improve scheduler's performance. Scheduler always tries to find
+	// at least "minFeasibleNodesToFind" feasible nodes no matter what the value of this flag is.
+	// Example: if the cluster size is 500 nodes and the value of this flag is 30,
+	// then scheduler stops finding further feasible nodes once it finds 150 feasible ones.
+	// When the field is unset or the value is 0, the PercentageOfNodesToScore in the top-level
+	// KubeSchedulerConfiguration will be used.
+	// Note: This field will override the top-level PercentageOfNodesToScore when it is not zero.
+	PercentageOfNodesToScore *int32 `json:"percentageOfNodesToScore,omitempty"`
```

Comment:
@liggitt do we have precedence on defaulting a field based on another? in this case, this is a per-profile field that when not set should be defaulted to the component configuration level field above (which we are planning to remove in v1beta2).